.gitignore

```diff
@@ -31,6 +31,9 @@ yarn-error.log*
 # vercel
 .vercel
 
+# vscode
+.vscode
+
```

Comment:
easy one to just remove this now if you want as it's happening anyway in https://github.com/icgc-argo/rdpc-ui/pull/20/commits/c56b0f101b6b727aa43d8be858784bee00591011

Reply:
This I added back based on team input: https://github.com/icgc-argo/rdpc-ui/pull/20/files/2a47e297bd3f77e7c42737170fd9684da9b8fff9#r1235378846

Reply:
Yes - I just mean it's in the .gitignore here

Reply:
Oh my bad I see now